simulation/test_torrent_status.cpp

```diff
@@ -41,11 +41,161 @@ using namespace sim;
 namespace lt = libtorrent;
 
 // this is a test for torrent_status time counters are correct
-TORRENT_TEST(status_timers)
+// it is a long running test, we did already try to improve it:
+// - less slow peers vs many fast peers
+// - slow peers with smal torrent vs fast peers with large torrent
+// - less peers with large torrents vs more peers with smal torrent
+// - connection limit vs parallel requests
+// as conlusion the following combination take the least amount of
+// execution time per tick:
+// - many slow peers with small torrent and connection limit
+TORRENT_TEST(status_timers_time_shift_with_paused_torrent)
```

Comment:
since it takes a minute to run, I think this test should be disabled for now. I can take a look at speeding it up later